hack/testing/logging.sh

```diff
@@ -441,9 +470,30 @@ function removeAdminCert() {
   os::cmd::expect_success "oc patch secret logging-elasticsearch -p '{\"data\":{\"admin-cert\": null}}'"
 }
 
+function rebuildVersion() {
```

Comment:
Right.  Look at https://github.com/openshift/origin-aggregated-logging/blob/master/hack/testing/logging.sh#L124 - we have to build from local source because that's what the testing CI does - it checks out the master source locally on the jenkins machine, merges it with the code from the PR, and copies it to the test machine.  So there isn't a github or source repo with the actual code being tested - it only really exists on the local machine - so we have to do `oc start-build --from-dir=/path/to/origin-aggregated-logging`

There should probably be a shell function to do builds which should check for `$USE_LOCAL_SOURCE` - if true, do `oc start-build --from-dir=/path/to/origin-aggregated-logging` - and then have no direct calls to start-build in logging.sh.